recipes/clair3/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "clair3" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: http://www.bio8.cs.hku.hk/clair3/bioconda/Clair3_v{{ version }}.zip
+  sha256: bc3a0aaf38f034a559b22ba7c8d29a16ebe62d810cf7903d6a7520ad93341106
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - python=3.6.10
+    - pypy3.6
+    - numpy=1.19.5
+    - tensorflow=2.2.0
+    - pytables=3.6.1
+    - samtools=1.10
+    - whatshap=1.0
+    - pigz=2.4
+    - parallel=20191122
+    - zstd =1.4.4
+test:
+  imports:
+    - tensorflow
+  commands:
+    - which clair3.py
```

Comment:
There's no `clair3.py --version` or something like that?

Reply:
Thanks for your suggestions, I will add a version check